src/client/components/features/Formation/Consulter/Statistiques/StatistiquesFormation.test.tsx

```diff
@@ -0,0 +1,149 @@
+/**
+ * @jest-environment jsdom
+ */
+import {
+	render,
+	screen,
+	within,
+} from '@testing-library/react';
+
+import { StatistiquesFormation } from '~/client/components/features/Formation/Consulter/Statistiques/StatistiquesFormation';
+import { Statistique } from '~/server/formations/domain/statistique';
+
+const statistiques = (override?: Partial<Statistique>) :Statistique => {
+	return {
+		millesime: '2020-2021',
+		region: 'Pays de la Loire',
+		tauxAutres6Mois: '12',
+		tauxEnEmploi6Mois: '36',
+		tauxEnFormation: '22',
+		...override,
+	};
+};
+describe('StatistiquesFormation', () => {
+	describe('quand on reçoit des statistiques', () => {
+
+		it('affiche l’entête', () => {
+			render(<StatistiquesFormation statistiques={statistiques()}/>);
+
+			const entête =  screen.getByRole('heading', { level: 2 });
+			expect(entête).toHaveTextContent('Et après la formation ?');
+			expect(entête).toBeVisible();
+
+			const descriptionEntTête = screen.getByText('Découvrez les chiffres-clés liés à cette formation pour la région Pays de la Loire');
+			expect(descriptionEntTête).toBeVisible();
+		});
+
+		it('affiche les informations de statistiques', () => {
+			render(<StatistiquesFormation statistiques={statistiques()}/>);
+
+			const listeDeStatistique = screen.getByRole('list', { name: 'statistiques' });
+			expect(listeDeStatistique).toBeInTheDocument();
+		});
```

Comment:
On pourrait vérifier la présence des informations dans le test avec quelque chose type : 

```suggestion
		it('affiche les informations de statistiques', () => {
			render(<StatistiquesFormation statistiques={statistiques({
				millesime: '2019-2020',
				tauxAutres6Mois: '42',
				tauxEnEmploi6Mois: '34',
				tauxEnFormation: '24',
			})}/>);

			const listeDeStatistique = screen.getByRole('list', { name: 'statistiques' });

			expect(listeDeStatistique).toHaveTextContent('34% sont en emploi au bout de 6 mois');
			expect(listeDeStatistique).toHaveTextContent('24% sont inscrits en formation');
			expect(listeDeStatistique).toHaveTextContent('42% sont dans d’autres cas');
			expect(listeDeStatistique).toHaveTextContent('promotion 2019-2020');
		});
```